modules/gtk/hm.nix

```
@@ -5,24 +5,11 @@
   options,
   ...
 }:
-
-let
-  cfg = config.stylix.targets.gtk;
-
-  baseCss = config.lib.stylix.colors {
-    template = ./gtk.css.mustache;
-    extension = ".css";
-  };
-
-  finalCss = pkgs.runCommandLocal "gtk.css" { } ''
-    cat ${baseCss} >>$out
-    echo ${lib.escapeShellArg cfg.extraCss} >>$out
-  '';
-in
 {
```

You forgot importing and calling `mkTarget`:

```suggestion
mkTarget {
```

The module arguments may have to be adapted:

https://github.com/nix-community/stylix/blob/5259682ce58d935f248297bf1c9793a5cee0787e/modules/gtk/hm.nix#L1-L7